src/Cinnabari.php

```diff
@@ -39,10 +39,16 @@ public function __construct(Schema $schema)
     // TODO: trim the query string before Cinnabari
     public function translate($query, $arguments)
     {
-        $tokens = self::getTokens($query);
-        $request = self::getRequest($tokens);
+        try {
+            $tokens = self::getTokens($query);
+            $request = self::getRequest($tokens);
+
+            return self::getResult($this->schema, $request, $arguments);
+        } catch (Exception $exception) {
+            echo $exception->getMessage() . "\n";
```

Comment:
Since this is the top level of the Cinnabari project, I thought it made sense to end the hierarchy of exceptions here. But I suppose if people were to include Cinnabari in their own projects, they might prefer to handle the exception for themselves. I based this echo statement off of the one a couple dozen lines down, which was already in there before I edited the file.